tools/test-generator/templates/bracket-push/template.ml

```diff
@@ -0,0 +1,16 @@
+open Core.Std
```

Comment:
It's been quite fun to work on, and I've learnt a fair bit about Ocaml development in the process. I'm very grateful to Exercism for that.